src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -64,6 +60,10 @@
         .withName(OFFSET_PARAM)
         .withDefaultValue(String.valueOf(DEFAULT_OFFSET))
         .withDescription("If specified, the number of records to skip/offset.").build();
+    public static final ParameterDescriptor USER_ID = ParameterDescriptor.newBuilder()
+        .withName(USER_ID_PARAM)
+        .withRequired(false)
+        .withDescription("If specified, the required user id.").build();
```

Comment:
Description's use of the word "required" is somewhat confusing since this param is marked as not required. It does raise the question though of whether the user id should be required. We initially opted to identify the user based on the Auth token, but since we've moved away from that, maybe we should require it for more methods to disambiguate things. Let's hold off for now, but could you create an issue for this?